src/languages/objectivec.js

```diff
@@ -69,16 +69,24 @@ function(hljs) {
       },
       {
         className: 'meta',
-        begin: '#',
-        end: '$',
+        begin: /#\s*[a-z]+\b/, end: /$/,
```

Comment:
No macros with underscores?

Reply:
(I just grabbed this from the cpp language definition.)

But anyway the token immediately after the `#` isn't a macro but a preprocessing instruction like `if` or `endif` or `pragma`. I don't believe any of them have underscores. I'm not sure if this change is necessary, but since it's the same preprocessor that runs over C, C++, and Obj-C (and Obj-C++), I thought it'd be best to align the two.

Reply:
git blame says it dates to 1bc269c8668786d5c3ea06bff14826543797fe71 and de1718e41a7cdf292898efd1adf03de113ae3556, but it's not clear why the stricter matching on #.

Reply:
Right I know what it is. I’m just mislabeled it. Lol. Thanks for clarifying though.